pointmatcher/DataPointsFilters/MaxPointCount.cpp

```diff
@@ -66,35 +67,20 @@ MaxPointCountDataPointsFilter<T>::filter(const DataPoints& input)
 template<typename T>
 void MaxPointCountDataPointsFilter<T>::inPlaceFilter(DataPoints& cloud)
 {
-	const size_t N = static_cast<size_t> (cloud.features.cols() - 1);
-	
-	if (maxCount <= N) 
+	const auto N = static_cast<size_t>(cloud.features.cols() - 1);
```

Comment:
```suggestion
	const size_t N{static_cast<size_t>(cloud.features.cols() - 1)};
```

If you use a known type with bracket initialization you can benefit from the compiler informing you in case there are narrowing casts 
https://stackoverflow.com/questions/42975620/in-c-can-we-use-for-c-style-casting